library/L2_HAL/sensors/battery/coulomb_counter.hpp

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+namespace sjsu
+{
+// Abstraction Interface for a coulomb counter. This device can give us
+// information about a connected battery's power level.
+class CoulombCounter
```

Comment:
I am getting an error from doxygen saying that this isn't documented. Would you have any suggestions? See below.
![image](https://user-images.githubusercontent.com/36345325/68061264-36d00180-fcc1-11e9-93cf-49085cbad493.png)

Reply:
I see that i used two `//` instead of three `///` will try that.